train_settings:
  seed: 40
  use_checkpoints: true
  load_model: false
  timesteps_before_training: 25000
  eval_freq: 5000
  eval_eps: 10
  max_timesteps: 5000000
  max_episode_steps: 500
  base_file_name: "td7_agent"
